assets/javascripts/modules/mma/updatePayment.jsx

```diff
@@ -54,11 +54,12 @@ const WAITING = 'waiting'
 const SUCCESS = 'success'
 const FAILURE = 'failure'
 const FORM = 'form'
+const UNAVAILABLE = 'unavailable'
 
 const CardUpdate = ({ card, handler }) => (<div><button className="button button--primary button--large" onClick={handler}>•••• •••• •••• {card && card.last4 || '••••'} Update Card</button></div>)
 const Success = () => (<p>Thank you, we have successfully updated your payment details.</p>)
 const Failure = ({ phone }) => <p>Unfortunately, we are unable to update your payment details at this time, please contact the call centre. {phone}</p>
```

Comment:
Perhaps this should ask the users to try again, since we failed whilst actually processing the update?